myreadinglist/mail.py

```diff
@@ -47,3 +45,7 @@ def send_email(to_email, subject, body, from_email=FROM_EMAIL, html=True):
         )
 
     return response
+
+
+if __name__ == '__main__':
+    send_email('test-email@gmail.com', 'my subject', 'my message')
```

Comment:
just for ideas, here you could add `argparse` so we can call it as standalone script with say `-s subject -m message`, but leave as is, we don't need that for now

Reply:
Cool, happy to do this anytime